### ISSUE TITLE
SWD metric

### ISSUE THREAD

**Issue by fnzhan** [microsoft/CoCosNet#25]
would you mind share the repo for the SWD metric evaluation? I use this repo https://github.com/koshian2/swd-pytorch to evaluate the results, while the swd score of the pre-trained model is quite different from the paper (112.4 on ADE20K). I think it is caused by the different repo of SWD. @panzhang0212

**Comment by panzhang0212** [comment]
The swd is from the official PGGAN repo https://github.com/tkarras/progressive_growing_of_gans.

you can down [evaluation.zip](https://github.com/microsoft/CoCosNet/files/5468583/evaluation.zip) and run
python cal_sliced_wasserstein.py synthesis_folder real_folder

**Comment by fnzhan** [comment]
> The swd is from the official PGGAN repo https://github.com/tkarras/progressive_growing_of_gans.
> 
> you can down [evaluation.zip](https://github.com/microsoft/CoCosNet/files/5468583/evaluation.zip) and run
> python cal_sliced_wasserstein.py synthesis_folder real_folder

Thanks for your help!
